packages/liveblocks-yjs/README.md

```diff
@@ -0,0 +1,46 @@
+<p align="center">
+  <a href="https://liveblocks.io#gh-light-mode-only">
+    <img src="https://raw.githubusercontent.com/liveblocks/liveblocks/main/.github/assets/header-light.svg" alt="Liveblocks" />
+  </a>
+  <a href="https://liveblocks.io#gh-dark-mode-only">
+    <img src="https://raw.githubusercontent.com/liveblocks/liveblocks/main/.github/assets/header-dark.svg" alt="Liveblocks" />
+  </a>
+</p>
+
+# `@liveblocks/yjs`
+
+Provides YJS integration to effortlessly back your YJS apps with Liveblocks
```

Comment:
Same here, the product name is spelled "Yjs", not "YJS". Might be nice to update spelling consistently in all public docs.